### PR TITLE
feat: Instant Project Online

### PR DESCRIPTION
**Pull request by avparadox**
### PR Fixes:
-  This PR helps new developers run the project instantly.

### Changes:
- 1 Added a setup.sh file in the main file directory
- 2 Modified the .env.example so that setup.sh file works properly
- 3 Made huge changes to the  Readme file so that users can fork & run the project faster

### Checklist before requesting a review
- [X] I have performed a self-review of my code
- [X] I assure there is no similar/duplicate pull request regarding same issue

Hey Sargam, firstly thanks for looking up on my PR. This is my first PR of life. Please feel free to scold me if my PR have any issue with respect to Open Source Contributing rules. I would love to improve & learn from you. 

Thanks for providing your valuable time & reviewing my PR.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -2,7 +2,7 @@ NEXT_PUBLIC_BASE_URL_LOCAL=http://127.0.0.1:3000
 ADMIN_SECRET="ADMIN_SECRET"
 JWT_SECRET="JWT_SECRET"
 # DONT CHANGE FOR RUNNING WITH DOCKER
-DATABASE_URL="postgresql://postgres:postgres@db:5432/cms?schema=public"
+DATABASE_URL="postgresql://myuser:mypassword@localhost:5432/mydatabase?schema=public"
```
Comment (devsargam) [inline .env.example:5]:
```suggestion
DATABASE_URL="postgresql://myuser:mypassword@localhost:5432/cms?schema=public"
```

File: README.md [hunk 2]
```diff
@@ -18,8 +18,24 @@ git clone https://github.com/code100x/cms.git
...
+> Your Docker Demon should be online
+
+1. Running Script for Instant setup
+
+```
+# Gives permission to execute a setup file
+chmod +x setup.sh
+
+# Runs the setup script file
+./setup.h
```
Comment (devsargam) [inline README.md:33]:
```suggestion
./setup.sh
```

File: README.md [hunk 3]
```diff
@@ -18,8 +18,24 @@ git clone https://github.com/code100x/cms.git
...
+```
+
+# Traditional Docker Setup
+
+   (Optional) Start a PostgreSQL database using Docker:
```
Comment (devsargam) [inline README.md:38]:
```suggestion
(Optional) Start a PostgreSQL database using Docker:
```

**Review by devsargam — APPROVED** [review]

**Comment by devsargam** [comment]
Thank you sir!

**Comment by avparadox** [comment]
Thank you @devsargam for accepting my life's first PR. Your guidance 5 months back made this possible. Thank you very much.

**Pull request merged**